packages/peregrine/lib/talons/CartPage/ProductListing/useProduct.js

```diff
@@ -159,7 +181,11 @@ const flattenProduct = item => {
         url_key: urlKey,
         url_suffix: urlSuffix
     } = product;
-    const { url: image } = small_image;
+    const { url: image } =
+        (configurableThumbnailSource === 'itself' &&
+            configured_variant &&
+            configured_variant.small_image) ||
```

Comment:
If I am accurate, we have stopped doing such logical operations. Instead, the best would be to use the ternary operator.

```
(configurableThumbnailSource === 'itself' && configured_variant) ?
            configured_variant.small_image : small_image;
```

Reply:
Changed to ternary operator.
Thank you